src/Aquifer.API/Endpoints/Users/GetSelf/Endpoint.cs

```diff
@@ -20,9 +23,15 @@ public override async Task HandleAsync(CancellationToken ct)
             Id = user.Id,
             Name = $"{user.FirstName} {user.LastName}",
             Permissions = permissions,
-            Company = new CompanyResponse { Id = user.CompanyId }
+            Company = new CompanyResponse { Id = user.CompanyId },
+            LanguageId = user.LanguageId,
         };
 
+        if (userService.HasPermission(PermissionName.CreateCommunityContent)) {
```

Comment:
Similar to create translation, I think the absence of the `CreateContent` permission should play a part here. I would keep the existing check for `CreateCommunityContent` but also add `&& !userService.HasPermission(PermissionName.CreateContent`.